example/test/react_test_components.dart

```diff
@@ -7,9 +7,10 @@ import "package:react/react_dom.dart" as react_dom;
 class _HelloComponent extends react.Component2 {
   @override
   get propTypes => {
-        'name': (Map props, propName, componentName, location, propFullName) {
-          if (props[propName].length > 20) {
-            return ArgumentError('(${props[propName]}) is too long. $propName has a max length of 20 characters.');
+        'name': (Map props, info) {
+          String propValue = props[info.propName];
+          if (propValue.length > 20) {
+            return ArgumentError('($propValue) is too long. $propValue has a max length of 20 characters.');
           }
           return null;
```

Comment:
`return null` is no longer part of the boilerplate
```suggestion
```

Reply:
![nope](https://user-images.githubusercontent.com/1750797/68408189-25b04600-0142-11ea-8766-ad1a59e16154.gif)

Reply:
😿